src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/InputPlaybackActionData.cs

```diff
@@ -42,6 +44,39 @@ public void StartAction(int segmentNumber, Dictionary<long, ObjectStatus> curren
             }
         }
 
+        public void PauseAction(int segmentNumber)
+        {
+            pauseTime = Time.unscaledTime;
```

Comment:
correct.. this is for avoiding the situation that you pause for a minute.. then hit resume and all the actions try to play instantly instead of resuming at their proper timing gaps